### ISSUE TITLE
Pull ortholog interacting data set for Fanconi demonstrator

### ISSUE THREAD

**Issue by mellybelly** [NCATS-Tangerine/ncats-ingest#20]
We need to pull a series of data to inform which genes to look for variants in. 

There is a google doc here for reference, 
https://docs.google.com/spreadsheets/d/1yX-5sfrC3vrahf4_k7-5rl4Oqzm853ollIMmUo1PTc0/edit#gid=1185309083

but essentially we need a gene set, **from set 4(n) in the spreadsheet:**

a) pull orthologs of the 21 FA genes (some may have alternate primary symbols): 
FANCA, FANCB, FANCC, FANCE, FANCF, FANCG, FANCL, FANCM, FANCD2, FANCI, UBE2T
FANCD1 (BRCA2), FANCJ, FANCN, FANCO, FANCP, FANCQ, FANCR, FANCS, FANCV, FANCU
FAAP100, FAAP24, FAAP20, FAAP16 (MHF1), FAAP10 (MHF2)

b) all the interaction partners of any of these orthologs from BioGrid and/o other source (maybe we can query STRING or ??

c) return the human orthologs of these interaction parters

@dnahotline will advise as needed

**Comment by jmcmurry** [comment]
@kshefchek has pulled in experimental protein-protein interaction data from STRING.

**Comment by mbrush** [comment]
Does the ingest of BioGrid and of STRING 'experimental' interactions suffice for the Translator St Jude's use case?  Or do we want to pull additional data here -  e.g. other types of 'interactions' in the STRING db, or interactions from other protein interaction knowledge sources?

**Comment by mellybelly** [comment]
Yes I think that is fine for now, though genetic interactions would be nice too, there would be more pressing things
